client/src/actions/auth.js

```diff
@@ -39,17 +39,16 @@ export const loadUser = () => async dispatch => {
 
 // Register User
 
-export const register = ({ username, password }) => async dispatch => {
+export const register = (registerData) => async dispatch => {
```

Comment:
Of course, it's cleaner, but I bet you will forget what kind of data we expecting in 1 week.